packages/cli-dashboard/src/components/siteReport/tabs/cookies/cookiesLandingContainer/index.tsx

```diff
@@ -51,33 +46,12 @@ const CookiesLandingContainer = ({
           />
         </div>
       )}
-      <CookiesLanding
-        tabFrames={tabFrames}
-        tabCookies={tabCookies}
-        showInfoIcon={false}
-        showBlockedInfoIcon={true}
-        associatedCookiesCount={Object.values(tabFrames).length}
-        showMessageBoxBody={false}
-        showBlockedCookiesSection
-        cookieClassificationTitle="Categories"
-      >
-        <div className="flex flex-col">
-          <div className="pt-4">
-            <CookiesMatrix
-              tabCookies={affectedCookies}
-              componentData={
-                prepareCookieStatsComponents(
-                  prepareCookiesCount(affectedCookies)
-                ).legend
-              }
-              tabFrames={tabFrames}
-              description=""
-              showInfoIcon={false}
-              showHorizontalMatrix={false}
-              allowExpand={false}
-            />
-          </div>
-        </div>
+      <CookiesLanding>
```

Comment:
We have a missing BlockedCookiesSection here.
With blocked reasons and categories
<img width="1470" alt="Screenshot 2024-03-05 at 8 23 53 PM" src="https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/58820001/2f402a1b-9f6b-44e5-b76f-9475bcff113f">

Reply:
Fix in latest commit.